src/Components/Game/index.tsx

```diff
@@ -1,20 +1,31 @@
 import React, { useState } from "react";
 import ColorPalette from "../ColorPalette";
 import "./style.css"
-import {COLOR_LIST, INITIAL_GAME_STATE, TOTAL_ATTEMPTS} from "../Utils/constants";
+import { COLOR_LIST, INITIAL_GAME_STATE, PEGS_PER_ATTEMPT, TOTAL_ATTEMPTS } from "../Utils/constants";
 import AttemptsContainer from "../AttemptsContainer";
-import { initializeBoard, initializeBoardHints } from "../Utils/util";
+import { generateSolution, initializeBoard, initializeBoardHints } from "../Utils/util";
+import swal from 'sweetalert';
 
+const gameSolution = generateSolution();
+const initialBoadColors = initializeBoard();
+const initialBoardHints = initializeBoardHints();
```

Comment:
for better readability constants should be in capital letters

Reply:
As we discussed, these constants are just for saving the initial references for the states and we had to call the functions again with the spread operator to avoid referring to the same object again and again.
@Ishaan-flyfin Please let me know if I should be changing them or not.